rdrf/rdrf/templates/rdrf_cdes/patients_listing.html

```diff
@@ -216,14 +175,52 @@
           <span id="loading_text"><i>{% trans 'Loading...' %}</i></span>
       </p>
 
-      <table id="patients_table" class="table table-striped table-hover">
-          <thead>
-              <tr>
-                  {% for column in columns %}
-                      <th>{{ column.label|translate|safe}}</th>
+      <div class="row">
```

Comment:
Will this be able to accommodate filters that need more horizontal width?

We could make the outer container `container-fluid`, or drop it altogether and use these columns to give a bit more horizontal space on desktop.

Reply:
When I expand the horizontal width, the alignment between the form button (Add Patient) and the page content no longer aligns. This is due to how these elements are structured in the `base-1-cols` template. It's not an easy fix, I will potentially need to create a new base template to handle this. 

How would you like me to proceed?

![image](https://user-images.githubusercontent.com/60163872/174925198-c8642ae5-3357-4547-9e9b-8bde0482b148.png)

![image](https://user-images.githubusercontent.com/60163872/174925358-5dd9b580-4269-4b9d-8b18-efbe9cd01bd9.png)

Reply:
Looks like those buttons are also aligned because they're inside a `container` in both the 1-col and 2-col templates. Seems using an outer-level `container` is an established pattern in the templates that would need to be considered as a whole.

Should we just table this change until we need to accommodate more filters?

Reply:
Sounds good.